sklearn/preprocessing/imputation.py

```diff
@@ -102,11 +103,21 @@ class Imputer(BaseEstimator, TransformerMixin):
         - If `axis=0` and X is encoded as a CSR matrix;
         - If `axis=1` and X is encoded as a CSC matrix.
 
+    add_missing_indicator : boolean, optional (default=False)
+        If True, the transformed X will have an appended indicator matrix
+        to denote the features that have at least one missing value and
+        the indicator matrix has 1 for each missing value.
+
     Attributes
     ----------
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature if axis == 0.
 
+    imputed_features_ : array of shape (n_features_with_missing, )
+        The input features which have been imputed during transform
```

Comment:
The newline disappears in the HTML docs, so please add a `.` at the end of this sentence.